tests/test_future_behaviour.py

```diff
@@ -111,8 +111,21 @@ def test_problematic_file():
 print("abc")
 """)
 
+FUTURE_IMPORT_PAREN = ("""\
+from __future__ import (absolute_import, division, print_function)
+unicode("abc")
+""", """\
+from __future__ import (absolute_import, division, print_function)
+import six
+six.text_type("abc")
+"""
+)
+
 def test_future_import_as():
     check_on_input(*FUTURE_IMPORT_AS)
 
 def test_future_import_as_multiple():
     check_on_input(*FUTURE_IMPORT_AS_MULTIPLE)
+
+def test_future_import_paren():
+    check_on_input(*FUTURE_IMPORT_PAREN)
```

Comment:
Don't know if it matters, but there is a missing newline at the end of the file.